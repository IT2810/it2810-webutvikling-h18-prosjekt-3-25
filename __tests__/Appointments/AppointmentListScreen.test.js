import React from 'react';
import CreateAppointment from '../../components/Appointments/CreateAppointmentScreen.js';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';

it("renders correctly", () => {
    const tree = renderer.create(<CreateAppointment />).toJSON();
    expect(tree).toMatchSnapshot() ;
})
