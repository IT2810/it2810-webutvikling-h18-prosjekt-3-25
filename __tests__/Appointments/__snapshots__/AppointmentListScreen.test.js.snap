// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "borderBottomColor": "#87cefa",
          "borderBottomWidth": 1,
          "fontSize": 30,
          "justifyContent": "center",
          "textAlign": "center",
        }
      }
    >
      Appointments and Events
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-end",
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "margin": 7,
          },
          Object {
            "borderRadius": 54,
            "height": 104,
            "width": 104,
          },
          false,
          Object {
            "alignItems": "center",
            "backgroundColor": "#517fa4",
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 50,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î……
      </Text>
    </View>
  </View>
</View>
`;
